### PR TITLE
Use npm instead of yarn

### PR DESCRIPTION
**Pull request by rvdh**

### CODE REVIEW

**Comment by rvdh** [comment]
@ianpaschal Can you please have a look at the linting errors?

**Comment by ChrisTerBeke** [comment]
missing package-json.lock?

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "copyStylesheets": "copyfiles -f ./src/stylesheets/**/* ./dist/stylesheets/",
         "copyFonts": "copyfiles -f ./src/fonts/**/* ./dist/fonts/",
         "cleanDist": "rimraf dist",
-        "prepublishOnly": "yarn cleanDist; tsc; yarn copyImages; yarn copyStylesheets; yarn copyFonts",
+        "prepublishOnly": "npm run-script cleanDist; tsc; npm run-script copyImages; npm run-script copyStylesheets; npm run-script copyFonts",
```
Comment (ianpaschal) [inline package.json]: These can just say `npm run`

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -11,21 +11,21 @@ stages:
...
 
 test:
   stage: test
   script:
-    - yarn lint
-    - yarn test:coverage
-    - yarn build
+    - npm run-script lint
+    - npm run-script test:coverage
+    - npm run-script build
```
Comment (ianpaschal) [inline .gitlab-ci.yml]: `npm run` works fine.

**Comment by ianpaschal** [comment]
Yeah so this is what I mean by the fact that basically none of our repos were actually adhering to the linting for some reason. No idea how that's been the case for so long undetected but the point is that using NPM here is working fine, it's ESLint that got messed up.

**Comment by ianpaschal** [comment]
FIXXXXXXXXXDIT

**Comment by ianpaschal** [comment]
Fixed it using a little script to convert yarn.lock files into package-lock.json files. This doesn't include the new linting rules yet but it does mean we can get NPM working and then I can work out the linting on top of this.

**Pull request merged**